---
bool success
string message
string[] map_ids
